mountpoint-s3/src/fuse/session.rs

```diff
@@ -89,8 +92,29 @@ impl FuseSession {
     /// Block until the file system is unmounted or this process is interrupted via SIGTERM/SIGINT.
     /// When that happens, unmount the file system (if it hasn't been already unmounted).
     pub fn join(mut self) -> anyhow::Result<()> {
-        let msg = self.receiver.recv();
-        trace!("received message {msg:?}, closing filesystem session");
+        let mut sys =
+            System::new_with_specifics(RefreshKind::new().with_processes(ProcessRefreshKind::new().with_memory()));
+        let pid = Pid::from(process::id() as usize);
```

Comment:
Why not [sysinfo](https://docs.rs/sysinfo/latest/sysinfo/index.html)::[get_current_pid](https://docs.rs/sysinfo/latest/sysinfo/fn.get_current_pid.html#)?